content/Foundation/Summit/home.pml

```diff
@@ -284,10 +285,29 @@ Munich, Germany (hybrid), 10-13 Oct 2022
     For questions regarding location, venue, and registration, please contact <a title="sel4summit.cyber@hensoldt.net" href="mailto:sel4summit.cyber@hensoldt.net">sel4summit.cyber@hensoldt.net</a>.
   </p>
   <p>
-    For all other questions, please contact <a title="summit@sel4.systems"   href="mailto:summit@sel4.systems">summit@sel4.systems</a>.
+    For all other questions, please contact <a title="summit@sel4.systems" href="mailto:summit@sel4.systems">summit@sel4.systems</a>.
   </p>
 </div>
 
+<div class="summit_div">
```

Comment:
- Maybe switch this section with the contact (have the contact as last section).
- Add "We are grateful the following sponsors for their financial contributions to support the seL4 summit 2022" just below section title
- Add the name underneath the logo like in the membership page
- Have a way to identify that it's a Bronze sponsor. Since we only have one for now, I'd suggest have it underneath the name (like the "Founding member" in the membership page)